src/components/GoatCounter.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+import { usePathname } from 'next/navigation';
+import Script from 'next/script';
+import { useEffect, useState } from 'react';
+
+export function GoatCounter() {
+  const pathname = usePathname();
+  const [goatcounter, setGoatcounter] = useState<GoatCounter | null>(null);
+
+  useEffect(() => {
+    if (!goatcounter) {
+      return;
+    }
```

Comment:
can we queue the pathnames that we receive here?
eg: user enters our page --> navigates away --> goatcounter loads and we only receive the second page view

Reply:
Actually I don't think we need this. On chrome the request gets completes instantly. On firefox the count request doesn't somehow doesn't complete instantly but the visit is still tracked on their site.